src/ui/wxWidgets/PasswordPolicy.cpp

```diff
@@ -379,6 +380,14 @@ bool CPasswordPolicy::Verify()
     mess = _("Policy name is already in use");
     id = ID_POLICYNAME;
     retval = false;
+  } else if ((m_polname != m_oldpolname) &&
+             (
+               (m_polname == _(stringx2std(PolicyManager::DEFAULT_POLICYNAME))) ||
+               (m_polname == stringx2std(PolicyManager::DEFAULT_POLICYNAME))
```

Comment:
Maybe add a method `PolicyManager::IsDefault(name)`? Also, why do you need both comparisons here?